moped-editor/src/views/projects/projectView/ProjectTimeline.js

```diff
@@ -614,29 +616,33 @@ const ProjectTimeline = ({ refetch: refetchSummary }) => {
                     return updateProjectPhase({
                       variables: updatedPhaseObject,
                     })
-                      .then(
-                        () =>
-                          // If update to the phase object was to phase_name or current_phase
-                          // update the project's current_phase and current_status.
-                          currentPhaseChanged
-                            ? updateProjectStatus({
-                                variables: {
-                                  projectId: projectId,
-                                  projectUpdateInput: isCurrentPhase
-                                    ? mappedProjectUpdateInput
-                                    : // Note: Below will overwrite a project's current_status and
-                                      // current_phase if the phase name is changed and its not a
-                                      // current_phase
-                                      {
-                                        current_status: "active",
-                                        current_phase: "active",
-                                        // we don't have a phase id for active, since it is not an official phase
-                                        current_phase_id: 0,
-                                      },
-                                },
-                              })
-                            : true // No change in project, safely ignore
-                      )
+                      .then(() => {
+                        // if the phase being updated is toggled as the current phase
+                        // update moped_project with new current_phase, updating the status badge
+                        if (isCurrentPhase) {
```

Comment:
❤️ the if/else blocks - I think this is a lot easier to read than the chained ternaries